lib/vmpooler.rb

```diff
@@ -33,7 +33,7 @@ def self.config(filepath='vmpooler.yaml')
     parsed_config[:config]['vm_checktime'] ||= 15
     parsed_config[:config]['vm_lifetime']  ||= 24
 
-    if parsed_config[:graphite]['server']
+    if defined? parsed_config[:graphite]['server']
```

Comment:
I think you want `parsed_config.has_key?(:graphite)`. That will check that the `:graphite` key even exists in the config, which will be `false` in your case.